### ISSUE TITLE
How to dynamically see changes in the loss curve and accuracy curve?

### ISSUE THREAD

**Issue by luminary111** [JiahuiYu/generative_inpainting#377]
or how to see the loss curve and accuracy curve?
I read carefully the relevant code in the neuralgym trainer and logger sections

**Comment by JiahuiYu** [comment]
I don’t think there is any accuracy curve.

**Comment by luminary111** [comment]
> I don’t think there is any accuracy curve.
Thank you for your busy reply！！！
Q1：
I replaced the single GPU training function in the train code
In your neuralgym toolkit, mention "There are two types of trainer in neuralgym: primary trainer and secondary trainer. For primary trainer, tensorflow related instances and configurations will be initialized, eg init all variables, summary writer, session, start_queue_runner and others. For the secondary trainer only train_ops and losses are iteratively updated / ran. ''
I think the train function in the primary trainer executes a set of batch_size, and the secondary trainer takes the new batch_size from the pipeline to calculate,
Because _, loss_value = sess.run ([self._train_op, self._loss], feed_dict = self.context ['feed_dict']) will be executed in both train functions.
Is my understanding correct?
Q2：In your train.py, model.build_graph_with_losses (FLAGS, images, FLAGS, summary = True, reuse = True) in the function multigpu_graph_def (model, FLAGS, data, gpu_id = 0, loss_type = 'g'),
I think the third parameter Flags should be True or False, because in the definition, the accepted parameter is training = True
Q3 :Do g_loss and d_loss need to converge at the same time for training?
